cmd/main.go

```diff
@@ -166,50 +172,88 @@ func (o *Options) LoadConfiguration(b []byte) error {
 	return nil
 }
 
-func (o *Options) LoadConfigurationFile() error {
-	configPaths := []string{
-		filepath.Join("{CONFIG}", "kubectl-ai", "config.yaml"),
-		filepath.Join("{HOME}", ".config", "kubectl-ai", "config.yaml"),
-	}
+var defaultConfigPaths = []string{
+	"{CONFIG}/kubectl-ai/config.yaml",
+	"{HOME}/.config/kubectl-ai/config.yaml",
+}
 
-	for _, configPath := range configPaths {
-		// Try to load configuration
-		tokens := strings.Split(configPath, string(os.PathSeparator))
-		for i, token := range tokens {
-			if token == "{CONFIG}" {
-				configDir, err := os.UserConfigDir()
-				if err != nil {
-					return fmt.Errorf("getting user config directory: %w", err)
-				}
-				tokens[i] = configDir
+// expandConfigPath replaces {CONFIG} and {HOME} tokens in a path with actual directories.
+func expandConfigPath(configPath string) (string, error) {
+	tokens := strings.Split(configPath, "/")
```

Comment:
This is moving from `os.PathSeparator` --> `/` and will not work on windows. Pl. restore the change.

Reply:
[`23f4c35` (#244)](https://github.com/GoogleCloudPlatform/kubectl-ai/pull/244/commits/23f4c35cbee8c6c4cc733341d247dc902d78ebfc)